packages/libs/src/sdk/services/Solana/programs/RewardManagerClient/RewardManagerClient.ts

```diff
@@ -22,6 +33,61 @@ import {
   GetSubmittedAttestationsSchema
 } from './types'
 
+type CustomInstructionErrorMessage = {
+  InstructionError: [number, { Custom: number }]
+}
+
+/**
+ * Mapping of custom instruction error codes to error messages
+ * @see {@link https://github.com/AudiusProject/audius-protocol/blob/2a37bcff1bb1a82efdf187d1723b3457dc0dcb9b/solana-programs/reward-manager/program/src/error.rs solana-programs/reward-manager/program/src/errors.rs}
+ */
+const codeMessageMap: Record<RewardManagerErrorCode, string> = {
+  [RewardManagerErrorCode.IncorrectOwner]:
+    'Input account owner is not the program address',
+  [RewardManagerErrorCode.SignCollision]:
+    'Signature with an already met principal',
+  [RewardManagerErrorCode.WrongSigner]: 'Unexpected signer met',
+  [RewardManagerErrorCode.NotEnoughSigners]: "Isn't enough signers keys",
```

Comment:
Yes I know the grammar here is wrong. It's a direct 1:1 copy from the program itself, which outputs this in the program logs on chain.